SocketRocket/SRWebSocket.m

```diff
@@ -151,6 +153,8 @@ @implementation SRWebSocket {
     SRProxyConnect *_proxyConnect;
 }
 
+@synthesize readyState = _readyState;
```

Comment:
Creates an ivar, since both getter+setter are overridden and I would rather not have an explicit ivar added, but attach everything to properties.